shopinvader_cart_expiry/tests/test_cart_expiry.py

```diff
@@ -32,6 +39,28 @@ def test_cart_expiry_scheduler(self):
         self._check_nbr_job_created(1)
         return
 
+    def test_cart_expiration_date(self):
+        so_date = fields.Datetime.from_string(self.cart.write_date)
+        today = fields.Datetime.to_string(so_date + timedelta(hours=5))
+        self.backend.write(
+            {"cart_expiry_delay": 1, "cart_expiry_policy": "cancel"}
+        )
+        now_method = "odoo.fields.Datetime.now"
+        so_date = fields.Datetime.from_string(self.sale.write_date)
+        today = fields.Datetime.to_string(so_date + timedelta(hours=5))
+        self.backend.write(
+            {"cart_expiry_delay": 1, "cart_expiry_policy": "cancel"}
+        )
+        now_method = "odoo.fields.Datetime.now"
+        with mock.patch(now_method) as mock_now:
```

Comment:
would be better to use freezegun

Reply:
Needs to add a dependency just for a test ?

Reply:
well, the behavior w/ freezegun is not the same. You grant that whatever piece of code is involved is going to use the same date. Also, next time someone will add a test that requires a date patch, likely he/she's going to copy/paste this.
Anyway, not a blocker but TODO comment wouldn't hurt ;)

Reply:
I also like freezgun... TODO would be nice.

Reply:
another one https://github.com/shopinvader/odoo-shopinvader/pull/535#pullrequestreview-331231906 :)